### DISCUSSION TITLE
Building for iOS without a Apple Mac?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#12892] [General]
Hi is it possible to build and publish to the Apple App Store without access to an Apple Mac?

Thanks Ward

**Comment by jeromelaban** [comment]
Thanks for the question. It's not yet supported, we're tracking this with https://github.com/unoplatform/uno/issues/12388

**Reply by jeromelaban** [reply]
Hot restart support will be available soon through https://github.com/unoplatform/uno/pull/12894, and will allow for building apps without a map. Publishing to the store needs mac, or a Cloud CI to perform the build and publishing to the app store.

**Reply by olumide-oyetoke** [reply]
> Hot restart support will be available soon through #12894, and will allow for building apps without a map. Publishing to the store needs mac, or a Cloud CI to perform the build and publishing to the app store.

I guess the original request was for building of iOS apps on Windows. Not Hot Reloading of an existing app.

**Reply by MartinZikmund** [reply]
@olumide-oyetoke Hot Restart's purpose actually is building the iOS app on Windows and deploying to a iOS device - you can follow similar steps to the [docs from Microsoft here](https://learn.microsoft.com/en-us/dotnet/maui/ios/hot-restart?view=net-maui-9.0)

**Reply by olumide-oyetoke** [reply]
> @olumide-oyetoke Hot Restart's purpose actually is building the iOS app on Windows and deploying to a iOS device - you can follow similar steps to the [docs from Microsoft here](https://learn.microsoft.com/en-us/dotnet/maui/ios/hot-restart?view=net-maui-9.0)

The information is written in the link you provided.
"Hot restart isn't a replacement for the full build experience provided by a Mac build host. For example, it can only deploy apps that use the debug build configuration, and it doesn't support static libraries and frameworks, XCFrameworks, or binding resource packages."
